lib/ws-kernel.coffee

```diff
@@ -18,8 +18,14 @@ class WSKernel extends Kernel
     interrupt: ->
         @session.kernel.interrupt()
 
+    shutdown: (restart = false) ->
+        if restart
```

Comment:
The restart option in `@shutdown` for `ZMQKernel` is just a parameter in the [shutdown message](https://github.com/nteract/hydrogen/blob/master/lib/zmq-kernel.coffee#L119) and doesn't really restart the kernel. (I don't really know what it is doing instead, maybe @n-riesco knows.)

I think for `WSKernel` we don't need the restart option in `@shutdown`.